packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -27,8 +32,41 @@ class Avatar extends Widget<Properties> {
 
     super._initMarkup();
 
-    this._renderInitialsElement();
-    this._updateInitials();
+    this._renderAvatarContent();
+  }
+
+  _renderAvatarContent(): void {
+    this._renderImage();
+    this._renderInitials();
+  }
+
+  _renderImage(): void {
+    this._$image?.remove();
+
+    const { url } = this.option();
+
+    if (url) {
```

Comment:
Can we extract if conditions from '_renderImage' and '_renderInitials' into 'rencerAvatraContent' method? Seems, in this case it will more readable, something like this
```
if(shouldRenderInitials) {
     this._renderInitials();
} else {
    this._renderImage();
}
```

Reply:
Also seems we shouldn't save _$initials & _$image and easily to remove $element().content() and render it's again when it's need

Reply:
I get your basic idea, but I don't think it should be exactly like this: shouldRenderInitials. I don't think we should move the name property out of _renderInitials.

Reply:
[4da065f777adfedddf3bf741d68b4ba898273434]